packages/backend/src/clients/github/Github.ts

```diff
@@ -87,7 +87,7 @@ export const getLastCommit = async ({
     owner: string;
     repo: string;
     branch: string;
-    token: string;
+    token: string; // TODO use installationId instead, to act as a bot
```

Comment:
do we need to make this change here or irrelevant to this PR?

Reply:
Maybe later, this method is used in other places (custom metrics), so I don't want to refactor it now